src/custom/utils/environments.ts

```diff
@@ -4,14 +4,16 @@ export function checkEnvironment(host: string) {
   const domainDevRegex = getRegex(process.env.REACT_APP_DOMAIN_REGEX_DEV)
   const domainStagingRegex = getRegex(process.env.REACT_APP_DOMAIN_REGEX_STAGING)
   const domainProdRegex = getRegex(process.env.REACT_APP_DOMAIN_REGEX_PROD)
+  const domainEnsRegex = getRegex(process.env.REACT_APP_DOMAIN_REGEX_ENS)
 
   return {
     isDev: domainDevRegex?.test(host) || false,
     isStaging: domainStagingRegex?.test(host) || false,
-    isProd: domainProdRegex?.test(host) || false
+    isProd: domainProdRegex?.test(host) || false,
+    isEns: domainEnsRegex?.test(host) || false
```

Comment:
you can also be super cool and do 
```ts
!!domainEnsRegex?.test(host)
```

Reply:
I don't understand, why would it be useful? is not the same as `!isEns`?

Reply:
you don't need the `|| false`, it's like doing `Boolean(sth)`

Reply:
ahh, ok, you mean that. Yes, that would be alright

Reply:
Actually, I thought about this but didn't actually change it.

What about we make the flags dependent?
Because it'll always be true for only 1 of them at a time.

Something like: isProd = !isDev && regex.test() and so on.

Reply:
If we really want to model that it can be only one at a time, we can change this into a enum.